js/src/components/modals/KeyphrasesTable.js

```diff
@@ -40,7 +40,7 @@ class KeyphrasesTable extends Component {
 	transformTrendDataToChartPoints( trend ) {
 		const trendArray = trend.split( "," );
 
-		return trendArray.map( ( value, index ) => ( { x: index, y: value } ) );
```

Comment:
This was trying to assign a string where a number is expected, firing a warning in the console:
```
Warning: Failed prop type: Invalid prop `data[0].y` of type `string` supplied to `AreaChart`, expected `number`.
    in AreaChart (created by KeyphrasesTable)
    in KeyphrasesTable (created by RelatedKeyphraseModalContent)
…
```